Write numerical results section (#26)
MichalisPanayides/Thesis: chapters/05_numerical_results/03_what_if/main.tex

        1.7127 & 2.3415 & 4.0998 & 6.3265 & 8.6058 & 9.9716 & 8.9634 \\
        1.7127 & 2.1269 & 3.4930 & 5.4885 & 7.8353 & 9.7075 & 9.7322
    \end{bmatrix}
\end{equation*}

\begin{equation}
    R = 
    \begin{bmatrix}
        0.22 & 0.06 & 0.05 & 0.05 & 0.04 & 0.03 & 0.01 \\
        0.95 & 0.6  & 0.47 & 0.37 & 0.28 & 0.2  & 0.11 \\
        0.97 & 0.81 & 0.72 & 0.62 & 0.51 & 0.37 & 0.21 \\
        0.97 & 0.85 & 0.77 & 0.68 & 0.57 & 0.44 & 0.26 \\
        0.98 & 0.89 & 0.83 & 0.76 & 0.66 & 0.53 & 0.35 \\
        1.   & 0.95 & 0.91 & 0.87 & 0.8  & 0.7  & 0.52
    \end{bmatrix}
\end{equation}
\normalsize

Matrices \(A\) and \(B\) are the payoff matrices of the two hospitals and
can be used to get the Nash equilibrium of the game.
Using the Lemke-Howson algorithm, the following pure strategy Nash equilibrium
is found:

\begin{equation}
    \sigma^A = (0, 0, 0, 0, 1, 0), \quad
    \sigma^B = (0, 0, 0, 0, 0, 1, 0)
\end{equation}

The output of the Lemke-Howson algorithm indicates that the game is at a Nash
equilibrium when hospital \(A\) plays a strategy of \(T^A = 5\) and hospital
\(B\) plays a strategy of \(T^B = 6\).

> In fact, this is the only Nash equilibrium of the game.

Reviewer: How do you know this?

Author: All algorithms (Support enumeration, Lemke-Howson, Fictitious play, Asymmetric replicator dynamics) point to just that N.E. I think it's unlikely that there is another one

A similar outcome is obtained when using a learning algorithm.
Figure~\ref{fig:asymmetric_replicator_dynamics_example_2} shows the output of
the run of the asymmetric replicator dynamics algorithm.

\begin{figure}[H]
    \centering
    \includegraphics[width=\linewidth]{chapters/05_numerical_results/Bin/example_2/base_case.pdf}
    \caption{Example 2: Asymmetric replicator dynamics}
    \label{fig:asymmetric_replicator_dynamics_example_2}
\end{figure}

Asymmetric replicator dynamics converges to the same strategy as the Lemke-
Howson algorithm, which is the pair of strategies \((T^A = 5, T^B = 6)\).

Consider now the concept of the Price of Anarchy
(\(PoA\))~\cite{roughgarden2005selfish}.
\(PoA\) is a measure in game theory that is used to quantify the efficiency of

> the outcome of a game when players behave in a selfish way.

Reviewer: ```suggestion
the outcome of a game when players behave in a selfish way.
Examples of using this measure in a healthcare setting include~\cite{} %  TODO Reference a few of my papers and others
```

Reviewer: (This isn't to add citation of my work but to show that it's something somewhat established.)

Author: Yeah, yeah I understand. I think I'm just bad at citing 😅

> More specifically, the \(PoA\) measures the ratio between the worst possible
> outcome of a game and the best possible outcome of the game.

Reviewer: ```suggestion
More specifically, the \(PoA\) measures the ratio between the worst possible
equilibria outcome of a game (so that no players have an incentive to deviate) and the best possible centrally controlled outcome of the game (the best possible collective situation).
```

The \(PoA\) of a game is defined as:

\begin{equation}\label{eq:poa_definition}
    PoA = \frac{\max_{s \in E}{F(s)}}{\min_{s \in S}{F(s)}}
\end{equation}

where \(S\) is the set of all possible strategies of the players, \(E\) is the
set of all possible equilibria of the game and \(F(s)\) is a cost function to
measure the efficiency of when the players play strategy \(s\).
The \(PoA\) is a measure that is used to describe the overall efficiency of the
game, rather the independent efficiency of each player.

> For the purpose of this study a new concept is introduced that considers the
> ratio between each hospital's best achievable blocking time and the one that is
> being played.

Reviewer: ```suggestion
For the purpose of this study a measure is introduced that considers the
ratio between each hospital's best achievable blocking time and the one that is
being played.
```

> This new concept is defined as the compartmentalised price of anarchy of the

Reviewer: ```suggestion
This is defined as the compartmentalised price of anarchy of the
```

players of the game and is defined as \(PoA_i(s)\) where \(i \in \{A, B\}\) and
\(s \in S\) is the strategy played by player \(i\).

> The aim of this new measure is to quantify how inefficient a strategy can be.

Reviewer: ```suggestion
```

    \caption{Example 2: Asymmetric replicator dynamics and compartmentalised
    \(PoA\) with increased arrival of type 2 patients \(\lambda_2\)}
    \label{fig:poa_ard_example_2_lambda_2}
\end{figure}

While the strategies of the players don't change, the \(PoA\) of both players
actually decreases.
Increasing the arrival rate of type 2 patients (ambulance patients) shouldn't
make the game more efficient, but it does.
That is not because the blocking time is decreased.
As a matter of fact the blocking time of ambulances increases for both players.
The reason why the \(PoA\) decreases can be traced back to the definition of the
\(PoA\) from equation~\ref{eq:compartmentalised_poa_definition}.
\(PoA_i(s)\) is defined as ratio between the blocking time when strategy \(s\)
is played and the best achievable blocking time from all strategies.
Therefore, when the best achievable blocking time becomes worse, the \(PoA\) of
the players decreases, which is exactly what happens in this case.
Although this is an interesting outcome, it is not a sensible one.
Increasing the arrival rate of an already flooded system does not make the

> system more efficient.

Reviewer: This is similar to the findings of~\cite{}  % TODO Cite Komenda and Knight paper

A more appropriate way to increase the efficiency of the system is to increase

> the number of staff \(C\).

Reviewer: ```suggestion
the capacity \(C\).
```

Reviewer: I see what you mean.

The problem with `number of staff` is that's it's too precise. In practice this could be limited by beds, limited by a combination of nurse + doctor + ...

What about `service capacity`?

Author: In the generic version of the model I use `number of servers`. But I don't think this could be limited by beds. I think bed availability has to do more with the artificial hospital capacity `N`, right? 🤔

I will add a `TODO` for it to revisit at a later point

    \caption{Example 2: Asymmetric replicator dynamics and compartmentalised
    \(PoA\) with incentivisation.}
    \label{fig:poa_ard_example_2_penalty}
\end{figure}

It can be seen that the hospitals start out by playing their usual strategies
of \(T^A = 5\) and \(T^B = 6\).
After the penalised matrices \(\tilde{A}\) and \(\tilde{B}\) are introduced,
the hospitals start to play strategies of \(T^A = 6\) and \(T^B = 7\).
That has a similar effect to the ones of
Figure~\ref{fig:poa_ard_example_2_num_of_servers} but no additional members of
staff have been added.
Additionally, the \(PoA\) for both hospitals is decreased, indicating that
the overall blocking time is reduced.

The results of this example show that using careful incentivisation on a
managerial level can result in an improvement of the overall blocking time
for the hospitals.
Although, arguably by adding more staff, the hospitals could have achieved
a greater reduction in the blocking time, these results indicate that when
that is not possible or not desirable, incentivisation can be used to
achieve a similar effect.

> % TODO: Should there be more examples? Are these enough? Maybe investigate

Reviewer: No I think this fine.

> % TODO: Should there be more examples? Are these enough? Maybe investigate
> %       some more examples with different parameter sets.

Reviewer: ```suggestion
```
